web/src/user_card_popover.js

```diff
@@ -307,6 +308,23 @@ function render_user_card_popover(
         $popover_element[0],
         {
             placement: popover_placement,
+            popperOptions: {
+                modifiers: [
+                    {
+                        // Show popover at the middle of window on small widths.
+                        name: "offset",
+                        options: {
+                            offset: ({reference, _placement, popper}) => [
+                                0,
+                                window.innerWidth < media_breakpoints_num.md
```

Comment:
I think it would be really good to drop a comment in the code explaining what all is happening here.

Reply:
Done